sections/currentState.tex

```diff
@@ -0,0 +1,31 @@
+\section{Current state}
+
+As previously mentioned, \knox{} was initially started in 2020.
+The previous group chose to develop this layer using Java.
+
+Using the framework Apache Jena in combination with the Fuseki package, they set up a simple database system.
+
+The system has two functionalities. 
+Firstly, it can store knowledge graphs in HDT and convert triples to HDT.
+Secondly, it contains a prototype for counting the number of words in an article.
```

Comment:
```suggestion
First, it can store knowledge graphs in HDT and convert triples to HDT.
Second, it contains a prototype for counting the number of words in an article.
```